lib/core/Actions.ts

```diff
@@ -566,15 +607,16 @@ class Actions {
   /**
    * Covers search suggestion endpoints.
    *
-   * @param {string} client
-   * @param {string} query
-   * @returns {Promise.<Response>}
+   * @param client
+   * @param query
+   * 
    */
-  async getSearchSuggestions(client, query) {
+  async getSearchSuggestions(client: string, query: string): Promise<APIResponse<object>> {
     if (![ 'YOUTUBE', 'YTMUSIC' ].includes(client))
-      throw new Utils.InnertubeError('Invalid client', client);
+      throw new InnertubeError('Invalid client', client);
 
     const response = await ({
+      // NOTE: this is an instance of Request, so something is wrong
```

Comment:
A `Request` instance is just an Axios instance, hence why one can make request like that.